sr_moveit_planner_benchmarking/experiments/launch/benchmarking_parameters.launch

```diff
@@ -0,0 +1,16 @@
+<launch>
```

Comment:
this launch file is a duplicate of benchmark_scene_ground_with_boxes.launch

Reply:
We can remove this